tests/conftest.py

```diff
@@ -362,7 +363,7 @@ def search_clear(search):
     _search_create_indexes(current_search, current_search_client)
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture()
```

Comment:
minor: since they are dictionaries, can we make them scope `session` or even `module`? to avoid recreating them all the time (unless they are modified inside tests, but we use copy IIRC).

Reply:
https://github.com/inveniosoftware/invenio-rdm-records/issues/1196